scripts/sync_with_wildfly_swarm.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+SCRIPT_SRC="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd -P )"           # Script source directory
+TEMP_DIR="$(mktemp -d)"                                                     # Temporary directory, will be deleted
+
+REPO_NAME="wildfly-swarm"                                                   # Repository base name, and the directory name on the disk
+REPO_URL="https://github.com/wildfly-swarm/${REPO_NAME}.git"                # Repo Git URL
+REPO_BRANCH="master"                                                        # Branch to be synchronized
+
+FILES_LIST="${SCRIPT_SRC}/wildfly-swarm-files.txt"                          # File with file- or directory names to synchronize
+FILES_DESTINATION="$(realpath ${SCRIPT_SRC}/../docs/topics/wildfly-swarm)"  # Destination directory in this repository where to sync
+
+git rm $FILES_DESTINATION
+
+echo "Cloning the WildFly Swarm repository..." >&2
+
+set -e
+
+pushd $TEMP_DIR &>/dev/null
+git clone $REPO_URL --depth 1 --branch $REPO_BRANCH
+cd $REPO_NAME/docs
+mvn clean install -DskipTests -Dswarm.product.build
```

Comment:
Do we possibly want to have everything in community docs and `-Dswarm.product.build` in product docs? I'm fine with community docs documenting Swarm product, but it might be misleading (e.g. `bom-certified` doesn't exist in community Swarm).

Reply:
1) `swarm.product.build` is just a property, it's not a directory.
2) The upstream AsciiDoc sources are conditional to make sure we display the right content. See https://raw.githubusercontent.com/wildfly-swarm/wildfly-swarm/master/docs/howto/use-a-bom/index.adoc (look for `ifdef` and `ifndef`).

Reply:
Yea I know that Swarm can produce both upstream docs and product docs. The question is -- what version of Swarm docs do we include? Is it always product docs? (I'd guess yes.) Is that correct? (I'm not sure really.)

Reply:
The only docs where this matters are the fractions reference docs, and those are included only from the product build. All other docs are included without modification, i. e. unbuilt.

Reply:
That's not true, the explanation of BOMs is different between community Swarm and product Swarm too.

Reply:
As I said before, please refer to https://raw.githubusercontent.com/wildfly-swarm/wildfly-swarm/master/docs/howto/use-a-bom/index.adoc. Both the community descriptions and the product descriptions are already contained in the file, therefore we need not make any further modifications. Displaying only the product-relevant information is done through setting the `:product:` attribute in the master file at build time.

Reply:
OK, I still don't quite follow, but you know what you're doing, so I'll leave it at that and will file an issue later, if needed :-)

Reply:
Let's meet tomorrow if you have time, I'll explain. This is an important mechanic in maintaining and building the docs, so I'd like everybody involved to know exactly what's up. I'll send you an invite tomorrow.

Reply:
That makes sense. Thanks!